### PR TITLE
DOM Manipulation and Events: Added clarification to deprecated keyCode event

### PR DESCRIPTION
**Pull request by mohammadzfr**
Added clarification to the deprecated keyCode keyboard event for the first exercise.

## Because

 it isn't fully clear what has to be changed in order for the deprecated keyboard event to be removed

## This PR

- clarifies the value the `keyCode` property uses to register keyboard input
- defines the alternative value needed for the `code` property to function correctly

## Issue

Closes #XXXXX (not applicable in this case)

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [ ] If this PR addresses an open issue, it is linked in the `Issue` section
-   [ ] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [ ] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/javascript_basics/DOM_manipulation_and_events.md [hunk 1]
```diff
@@ -386,7 +386,7 @@ You can find a more complete list with explanations of each event on [this page]
...
+1.  Grab the first exercise in Wes Bos's JavaScript30 program by cloning the repo at [https://github.com/wesbos/JavaScript30](https://github.com/wesbos/JavaScript30). Code along with the [Video Tutorial](https://www.youtube.com/watch?v=VuN8qwZoego) to build the rest of the exercise. In the video you will notice that a deprecated [keycode](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) keyboard event is used, replace it with the recommended [code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) keyboard event. You will also need to replace the `data-key` tags accordingly (e.g. `65` in the [keycode](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode) event corresponds to `KeyA` in the [code](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) event).
```
Comment (rlmoser99) [inline foundations/javascript_basics/DOM_manipulation_and_events.md:389]: I find this wording slightly confusing and it links to the both resources twice, which I think adds to the confusion. Would it be possible to do something like this:
`... keyboard event and replace the `data-key` tags accordingly.`

**Review by rlmoser99 — CHANGES_REQUESTED** [review]
Thank you for taking the time to update this lesson. I left one comment with a change request.

File: foundations/javascript_basics/DOM_manipulation_and_events.md (hunk 1 above)
Comment (mohammadzfr) [reply foundations/javascript_basics/DOM_manipulation_and_events.md:389]: Thank you for the feedback, it sounds much better.

**Comment by mohammadzfr** [comment]
Pull request issues are resolved

**Review by rlmoser99 — APPROVED** [review]

**Pull request merged**